app/mstool/Parameters.cpp

```diff
@@ -63,6 +66,10 @@ void Parameters::parse_options(int argc, char* argv[])
     }
 
     logging::add_console(static_cast<severity_level>(log_level));
+    if (vm.count("log-file")) {
+        logging::add_file(log_file, static_cast<severity_level>(log_level));
+        L_(info) << "Logging output to " << log_file;
```

Comment:
I like the ideas of different severities. Trying to set severities my conclusions was: warn is not enough, info is too much (mainly because of the recurring status log). Maybe we should create a new severity between info and warn for status and log all recurring information to this.
Anyway I would do this in a new branch.